scripts/deploy_jenkins.sh

```diff
@@ -45,4 +45,8 @@ if [ "${NEXUS}" = true ]; then
    oc new-app -f  $TEMPLATES_DIR/nexus3-template.yaml
 fi
 
+if [ "${ENABLE_OAUTH}" = true ]
+  oc adm policy add-cluster-role-to-group view system:authenticated -n $PROJECT_NAME
```

Comment:
This gives view access to all openshift projects to any authenticated user which I don't think we want. Also, in reality we would need to give the user edit access since they cant trigger any jobs with just view.